packages/strapi-design-system/src/SimpleMenu/SimpleMenu.js

```diff
@@ -206,4 +211,5 @@ SimpleMenu.propTypes = {
   children: PropTypes.oneOfType([PropTypes.arrayOf(menuItemType), menuItemType]).isRequired,
   id: PropTypes.string,
   label: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.element]).isRequired,
+  size: PropTypes.string.isRequired,
```

Comment:
Can you the update proptype please -> `Proptypes.oneOf(['S', 'M']`.

Also, IMO this prop shouldn't be required. The default value should be set to M.

Reply:
Also, the `size` prop won't work is `as` is defined. Can you add some storybook description to this prop as specify that please. 

```js
/**
 * Here
*/
size: PropTypes.oneOf(['S', 'M'],
```

You can also add some description for all the props if you want 🙊

Reply:
Done in 851b710fc230a62fe933e426ecb4cb9a89f7121d